test/query.jl

```diff
@@ -252,8 +261,16 @@ try
         @fact isdefined(:LoadTest1) --> true # first module should load first but fail
         @fact x --> 42
         rm(fn)
-        close(rd);close(wr)
-        redirect_stderr(stderr_copy)
+    end
+    context("multiple errors") do
+        add_format(
+            format"MultiError",
+            (),
+            ".multierr",
+            [:MultiError1],
+            [:MultiError2]
+        )
+        @fact_throws ErrorException load("test.multierr")
```

Comment:
Cute fact: at least with the new `Base.Test` framework, it's easier than ever to check the specific error message thrown. Example: https://github.com/JuliaImages/ImagesFiltering.jl/blob/2269cc4af22a0977c9eb4db1bcc96038751ccf4c/test/border.jl#L57-L58